packages/sdk/src/contracts/prebuilt-implementations/marketplace.ts

```diff
@@ -438,6 +438,29 @@ export class Marketplace implements UpdateableNetwork {
     await this.contractWrapper.multiCall(encoded);
   }
 
+  /**
+   * returns the minimum bid a user can place to outbid the previous highest bid
+   * @param listingId - the listing id of the auction
+   */
+  public async getMinimumNextBid(
+    listingId: BigNumberish,
+  ): Promise<BigNumberish> {
+    const currentAuction = this.auction;
+    const currentBidBuffer = await this.getBidBufferBps();
```

Comment:
Hi @joaquim-verges, do you want to include moving the `getBidBufferBps()` function to `marketplace-auction.ts`? Because it's been originally in the `marketplace.ts` class and also, not mentioned as part of the ticket. I don't mind doing it though if moving things such as this will be allowed.

Reply:
yep, as mentioned above , i would move `getBidBufferBps()` into `MarketplaceAuction` but keep `Marketplace.getBidBufferBps()` exposed to avoid breaking the API.

The implementation of `Marketplace.getBidBufferBps()` would then just call into the implementation inside `MarketplaceAuction`

Reply:
hi @joaquim-verges, I made the change. Kindly check and let me know. Thanks much